content/en/ecosystem/distributions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Distributions
+description:
+  Libraries, services, and apps with first-party support for OpenTelemetry.
+aliases: [/distributions]
+---
+
+OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
+customizing OpenTelemetry components so that they're easier to deploy and use
+with specific observability back ends.
+
+Any third-party can customize OpenTelemetry components with back-end, vendor, or
+end-user specific changes. The following list contains a sample of OpenTelemetry
+distributions and their type.
+
+{{% ecosystem/distributions-table %}}
+
+## How to add your distribution {#how-to-add}
+
+To have your library, service, or app listed, submit a PR with an entry added to
+the
+[distributions list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/distributions.yaml).
+The entry must include the following:
+
+- Link to the main page of your distribution
+- Link to the documentation that explains how to use the distribution
```

Comment:
```suggestion
- Link to the main page of your distribution
- Link to the documentation that explains how to use the distribution
- List the components your distribution contains
- GitHub handle or email address as a point of contact so that we can reach out in case we have questions
```